### PR TITLE
20170406 net metric fixes

### PR DESCRIPTION
**Pull request by grahamwhaley**
Fix a typo error in a test, combine two near identical tests, and add some small comments to the file.

### CODE REVIEW

**Comment by GabyCT** [comment]
lgtm

**Comment by GabyCT** [comment]
@grahamwhaley thanks :)

File: tests/metrics/network/network-metrics.sh [hunk 1]
```diff
@@ -17,6 +17,10 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Test inter (docker<->docker) netwrok bandwidths and jitters
```
Comment (jodh-intel) [inline tests/metrics/network/network-metrics.sh]: typo: `netwrok`.

File: tests/metrics/network/network-metrics.sh [hunk 2]
```diff
@@ -58,40 +64,33 @@ function jitter {
...
 
+# Test TCP docker->docker bandwidth with multiple network connections (parallelism)
 function bandwidth_multiple_tcp_connections {
         setup
         number_tcp_connections=8
         multiple_tcp_result=$(mktemp)
         $DOCKER_EXE run -d --name=iperf-server ${image} bash -c "iperf -p ${port} -s" > /dev/null && \
 	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
-        $DOCKER_EXE run -ti --rm --name=iperf-client ${image} bash -c "iperf -c ${server_address} -P ${multiple_tcp_result} -t ${time}"  > "$multiple_tcp_result"
+        $DOCKER_EXE run -ti --rm --name=iperf-client ${image} bash -c "iperf -c ${server_address} -P ${number_tcp_connections} -t ${time}"  > "$multiple_tcp_result"
```
Comment (jodh-intel) [inline tests/metrics/network/network-metrics.sh]: Maybe not strictly part of this PR, but the indentation in this function looks inconsistent.

**Comment by grahamwhaley** [comment]
@dborquez Note that I've reworded some of the output strings to more accurately reflect client/server rather than local/remote - so that may affect your test data gathering filters?

File: tests/metrics/network/network-metrics.sh [hunk 3]
```diff
@@ -58,40 +64,33 @@ function jitter {
...
+# Test TCP bandwidth bi-directionally (docker_iperf_server<->docker_iperf_client)
+# Extract bandwidth results for both directions from the one test
+function bidirectional_bandwidth_server_client {
         setup
         bidirectional_bandwidth_result=$(mktemp)
         $DOCKER_EXE run -d --name=iperf-server ${image} bash -c "iperf -p ${port} -s" > /dev/null && \
 	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
         $DOCKER_EXE run -ti --rm --name=iperf-client ${image} bash -c "iperf -c ${server_address} -d -t ${time}" > "$bidirectional_bandwidth_result"
-        total_bidirectional_bandwidth=`cat $bidirectional_bandwidth_result | tail -1 | awk '{print $(NF-1), $NF}'`
+        total_bidirectional_server_bandwidth=`cat $bidirectional_bandwidth_result | tail -1 | awk '{print $(NF-1), $NF}'`
```
Comment (jodh-intel) [inline tests/metrics/network/network-metrics.sh]: Same comment on indentation.

**Comment by grahamwhaley** [comment]
@jodh-intel np - I'll hit the typo and have a look at the indentation at the same time.
expect v2 soon

File: tests/metrics/network/network-metrics.sh (hunk 3 above)
Comment (grahamwhaley) [inline tests/metrics/network/network-metrics.sh]: looks like a mix of tabs and spaces.... will fix them

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by amshinde** [comment]
lgtm

**Pull request merged**